x/tradeshield/types/events.go

```diff
@@ -0,0 +1,27 @@
+package types
+
+import (
+	"strconv"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	
+)
+
+const (
+	TypeEvtCloseOrder   = "close_order"
```

Comment:
@cryptokage1996 can we make the event name more specific, here we are dealing with `close_spot_order` thanks